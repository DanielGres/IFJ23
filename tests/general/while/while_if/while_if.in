// DOBRE
/* OUTPUT:
    c: 15
    a: 1
    a: 4
    a: 5
*/
var a = 0
while (a < 5) {
    var b = a + 5
    if (b == 5) {
        var c = 15
        write("c:", c)
    } else {
        a = a + 3
        write("a: ", a)
    }
    a = a + 1
    write("a: ", a)
}