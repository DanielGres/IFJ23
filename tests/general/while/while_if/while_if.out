c: 15
a: 1
a: 4
a: 5